base/util_testing.go

```diff
@@ -654,3 +359,8 @@ func DirExists(filename string) bool {
 	}
 	return info.IsDir()
 }
+
+// getTestKeyNamespace returns a unique doc key namespace that can be prepended in tests.
+func getTestKeyNamespace() string {
```

Comment:
Is t.Name a simpler approach in most use cases for this?  Or have you seen cross-package issues?

Reply:
I think using t.Name is a good idea, and enforces the "test only" nature via `testing.T`. Changed.